docs/source/notes/randomness.rst

```diff
@@ -22,12 +21,25 @@ CPU and CUDA)
     import torch
     torch.manual_seed(0)
 
+There are some PyTorch functions that use CUDA functions that can be a source
+of non-determinism. One class of such CUDA functions are atomic operations,
+in particular :attr:`atomicAdd`, where the order of parallel additions to the
```

Comment:
I'm a bit confused. Isn't addition order-agnostic?

Reply:
Not in floating point land :)